docs/index.md

```diff
@@ -2,17 +2,19 @@
 # todo: internal
 ---
 
-# Prometheus 1.8
+# Prometheus 2.0
 
 Welcome to the documentation of the Prometheus server.
 
 The documentation is available alongside all the project documentation at
-[prometheus.io](https://prometheus.io/docs/prometheus/1.8/).
+[prometheus.io](https://prometheus.io/docs/prometheus/2.0/).
```

Comment:
None of the links at https://prometheus.io/docs/prometheus/2.0/ work right now and will continue not to work after this PR as far as I can tell. Not sure how this page is accessed to be honest.

Reply:
We should probably change this to latest and remove the number from above to avoid having to remember to update it.

@grobie FYI on issue on that page.